src/set_theory/zfc/ordinal.lean

```diff
@@ -26,32 +26,47 @@ Further development can be found on the branch `von_neumann_v2`.
   `set_theory/ordinal/arithmetic.lean`.
 -/
 
+universe u
+
+variables {x y z : Set.{u}}
```

Comment:
Is the universe variable necessary? I notice it's not used in the definition of `is_transitive`

Reply:
Yes, since otherwise Lean tries to give different universes to `x`, `y`, `z`, which doesn't work since set operations don't work with sets in different universes.